### PR TITLE
Feature/asset stats crw 354

### PR DESCRIPTION
**Pull request by AbdallaEmadEldinAbdalla**
[CRW-354](https://techxarya.atlassian.net/browse/CRW-354)

Done except the charts (backend not ready yet)

[CRW-354]: https://techxarya.atlassian.net/browse/CRW-354?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/app/asset/assetInformation.tsx [hunk 1]
```diff
@@ -3,6 +3,9 @@ import { Col, Row } from "../../shared/layout/flex";
 import { assetTimeseries } from "../../../utils/constants/assetTimeseries";
 import { useTranslation } from "next-i18next";
 import { useSelector } from "react-redux";
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
```
Comment (almujtabamohammed) [inline src/components/app/asset/assetInformation.tsx:7]: Please move this import to _app.tsx

File: src/components/app/asset/assetInformation.tsx [hunk 2]
```diff
@@ -128,6 +134,67 @@ const AssetInformationTab: FC<IAssetInformationTab> = ({ stats }) => {
...
+                <Col className="flex-1 justify-center gap-2">
+                  <p className="text-4xl font-bold text-orange-1 text-center sm:text-left">{coinstats?.premium ? '12.6' : coinstats?.portfolio_holdings}%</p>
+                  <p className="text-lg md:text-sm font-bold text-white text-center sm:text-left">{t('statsHolding')}</p>
+                </Col>
+              </Row>
+
+              {coinstats?.premium && <Row className="top-0 right-0 bottom-0 left-0 absolute items-center justify-center gap-4 px-5">
+                <svg width="19" height="14" viewBox="0 0 19 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.9391 5.53333L10.1904 0.560852C9.7823 -0.18679 8.71145 -0.186954 8.30307 0.560379L5.54681 5.53273L1.65117 3.08802L1.6466 3.08519C0.835375 2.58549 -0.221885 3.30958 0.0409489 4.27744L0.0412243 4.27845L2.18223 12.2227L2.18269 12.2244C2.36327 12.8872 2.96454 13.3372 3.63922 13.3372H14.8626C15.5497 13.3372 16.1395 12.8835 16.3191 12.2244L16.3196 12.2227L18.4608 4.27767C18.7153 3.34106 17.6794 2.55101 16.8464 3.09073L12.9391 5.53333Z" fill="#F7931A" />
+                </svg>
```
Comment (almujtabamohammed) [inline src/components/app/asset/assetInformation.tsx:158]: Extract csv to separate file

File: src/components/app/asset/assetInformation.tsx [hunk 3]
```diff
@@ -128,6 +134,67 @@ const AssetInformationTab: FC<IAssetInformationTab> = ({ stats }) => {
...
+
+                <Button className="bg-blue-3 h-12 md:px-4 rounded-lg text-bold text-white font-bold text-sm">
+                  {t('seeMore')}
+                </Button>
+              </Row>
+
+              {coinstats?.premium && <Row className="top-0 right-0 bottom-0 left-0 absolute items-center justify-center gap-4 px-5">
+                <svg width="19" height="14" viewBox="0 0 19 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd" d="M12.9391 5.53333L10.1904 0.560852C9.7823 -0.18679 8.71145 -0.186954 8.30307 0.560379L5.54681 5.53273L1.65117 3.08802L1.6466 3.08519C0.835375 2.58549 -0.221885 3.30958 0.0409489 4.27744L0.0412243 4.27845L2.18223 12.2227L2.18269 12.2244C2.36327 12.8872 2.96454 13.3372 3.63922 13.3372H14.8626C15.5497 13.3372 16.1395 12.8835 16.3191 12.2244L16.3196 12.2227L18.4608 4.27767C18.7153 3.34106 17.6794 2.55101 16.8464 3.09073L12.9391 5.53333Z" fill="#F7931A" />
+                </svg>
```
Comment (almujtabamohammed) [inline src/components/app/asset/assetInformation.tsx:187]: Extract svg

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]

**Comment by alex-greenbull** [comment]
Screenshot_arya-crypto-web-git-feature-asset-stats-crw-354-arya-crypto vercel appassetsymbolbtc_001426
 
 change orange to blue

File: src/components/app/asset/assetInformation.tsx [hunk 4]
```diff
@@ -133,6 +139,63 @@ const AssetInformationTab: FC<IAssetInformationTab> = ({ stats }) => {
...
+            <Col className="relative flex-1">
+              <Row className={clsx({ "blur-md": coinstats?.premium }, "bg-grey-3 px-6 py-4 rounded-lg gap-8 flex-col sm:flex-row sm:items w-full")}>
+                <Col>
+                  <CircularProgressbar text="" value={coinstats?.premium ? 12.6 : coinstats?.portfolio_holdings} className="max-w-28 h-28" strokeWidth={18} styles={buildStyles({
+                    pathColor: "#558AF2",
+                    strokeLinecap: 'butt',
+                  })} />
+                </Col>
+                <Col className="flex-1 justify-center gap-2">
+                  <p className="text-4xl font-bold text-blue-1 text-center sm:text-left">{coinstats?.premium ? '12.6' : coinstats?.portfolio_holdings}%</p>
```
Comment (almujtabamohammed) [inline src/components/app/asset/assetInformation.tsx:155]: This is causing an error
Error: Objects are not valid as a React child (found: object with keys {day, holding_count}). If you meant to render a collection of children, use an array instead.

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]

File: public/locales/en/asset.json [hunk 5]
```diff
@@ -64,5 +64,9 @@
     "select_provider": "Please select a provider to trade",
     "select_base": "Please select a base asset",
     "trade_error": "Error creating trade",
-    "cryptoProfitCalculator":"Crypto Profit Calculator"
+    "cryptoProfitCalculator": "Crypto Profit Calculator",
+    "insights": "Insights",
+    "statsHolding": "BTC holders in ARYA Crypto",
+    "todayTrades": "BTC trades today",
```
Comment (almujtabamohammed) [inline public/locales/en/asset.json:70]: This "BTC" should be dynamic based on the asset name

**Review by almujtabamohammed — APPROVED** [review]

**Pull request merged**